servicemanager.go

```diff
@@ -168,15 +168,24 @@ func newServiceManager(cfg *config.Config) (sm *serviceManager, err error) {
 		}
 	}
 
+	if sm.serviceMgr, err = servicemanager.New(cfg, sm.db); err != nil {
+		return sm, aoserrors.Wrap(err)
+	}
+
 	if sm.layerMgr, err = layermanager.New(cfg, sm.db); err != nil {
 		return sm, aoserrors.Wrap(err)
 	}
 
+	if err := sm.layerMgr.CheckLayersConsistency(); err != nil {
```

Comment:
I guess it should be reimplemented: layrmanager should check layer consistency on its own and remove only not consistent layers.